src/utils/sformat.fs

```diff
@@ -60,59 +60,59 @@ namespace Microsoft.FSharp.Text.StructuredFormat
 #endif
         | ActivePatternCase of string
         | ActivePatternResult of string
-        | Alias of string
-        | Class of string
-        | Union of string
-        | UnionCase of string
-        | Delegate of string
-        | Enum of string
-        | Event of string
+        | Alias of obj option * string
```

Comment:
Yeah, it's defined later. Maybe introduce a new type just for this? I feel myself (very) uncomfortable looking at type erasure.